azkaban-common/src/test/java/azkaban/utils/EmailerTest.java

```diff
@@ -67,16 +67,18 @@ public void setUp() throws Exception {
     }
 
 
-
-
     /**
-     * test send email
+     * (test send email)
+     * if you want to run this case,
+     * please make sure these variable{host,mailPort,password,receiveAddr} effective.
+     *
```

Comment:
I would prefer that the test to fail when these varaibles are not set or set incorrectly. But I am ok with the current implementation for now.

Maybe also mention that this test will currently succeed because email sending errors are caught. To verify email functionality is working, users need to manually verify that a real email is sent and received?